src/main/scala/org/apache/spark/shuffle/ucx/UcxShuffleTransport.scala

```diff
@@ -252,7 +228,7 @@ class UcxShuffleTransport(var ucxShuffleConf: UcxShuffleConf = null, var executo
 
   def unregisterShuffle(shuffleId: Int): Unit = {
     registeredBlocks.keysIterator.foreach(bid =>
-      if (bid.asInstanceOf[UcxShuffleBockId].shuffleId == shuffleId) {
+      if (bid.asInstanceOf[UcxShuffleBlockId].shuffleId == shuffleId) {
```

Comment:
Maybe you can use `getClass` here

Reply:
Not sure if I follow. `getClass` returns the class of a variable. `asInstanceOf` is type cast